### PR TITLE
Wrestlemania Metagame

### PR DESCRIPTION
**Pull request by HiTechFlufi**
The scripts for the upcoming Wrestlemania randomized meta

### CODE REVIEW

**Comment by ghost** [comment]
niceme.me no but seriously, p cool

**Comment by DeathlyPlays** [comment]
For the WWE meta there is no formats code for it nor a moves.js for its custom moves

**Comment by DeathlyPlays** [comment]
Also that didn't fix the emotes it just removed emote timestamps....

**Comment by DeathlyPlays** [comment]
This has severe formatting issues, I'll try to fix them but no guarantees.... seeing as I am stripped for time.

**Comment by DeathlyPlays** [comment]
Fixed your formatting~

**Comment by DeathlyPlays** [comment]
Now you just need 3 more Pokemon for it to randomize properly~

File: mods/wwe/moves.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			mirror: 1,
+		},
+		secondary: false,
+		category: "Physical",
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Body Slam", target);
+		},
+		basePower: 150,
+		pp: 1,
```
Comment (DeathlyPlays) [inline mods/wwe/moves.js]: Make this 0.625

**Review by DeathlyPlays — DISMISSED** [review]

**Pull request closed without merging**